test/button_test.js

```diff
@@ -120,19 +120,28 @@ describe("Add-on Functional Tests", function() {
     await window.setSize(currentSize.width, currentSize.height);
   });
 
-  // ADDON UNINSTALLED
-  it("should no longer trigger animation once uninstalled", async() => {
-    await utils.uninstallAddon(driver, addonId);
-    await utils.copyUrlBar(driver);
-    const { hasClass, hasColor } = await utils.testAnimation(driver);
-    assert(!hasClass && !hasColor);
-  });
+  // These tests uninstall the addon before and install the addon after.
+  // This lets us assume the addon is installed at the start of each test.
+  describe("Addon uninstall tests", () => {
+    beforeEach(async() => {
```

Comment:
Changing both `beforeEach` and `afterEach` to `before` and `after` will work. We need independence at the test suite level (`describe()`), not the test (`it()`) level.